packages/rightdown/src/core/config-reader.ts

```
@@ -0,0 +1,252 @@
+import { readFileSync, existsSync } from 'node:fs';
+import yaml from 'js-yaml';
+import { Result, success, failure, makeError, type AppError } from '@outfitter/contracts';
```

ESLint

[error] Imports "Result" are only used as type.

(@typescript-eslint/consistent-type-imports)